hacks_mbof/settings.py

```diff
@@ -184,8 +185,13 @@
             'formatter': 'simple',
         },
         'ocellus_handler': {
-            'class': 'logging.StreamHandler',
+            'level': getenv('DJANGO_LOGGING_LEVEL', 'DEBUG'),
+            'class': 'logging.handlers.TimedRotatingFileHandler',
             'formatter': 'ocellus_formatter',
+            'filename': os.path.join(PROJECT_ROOT, 'logs/' + getenv('HOSTNAME', 'localhost') + '.log'),
+            'when': 'midnight',
+            'interval': 1,
+            'backupCount': 60,
```

Comment:
`interval` and `backupCount` would benefit from comments.  I'm not certain what they do, so explanations here would be great.